### PR TITLE
Add bridge du2

### PR DESCRIPTION
**Pull request by isentropy**
adds to smart-contracts-init:

1. deploy Tokenbridge Arbitrary Message Bridge (using docker-in-docker)
2. deploy Tokenbridge ERC677 mediator over Arbitrary Message Bridge (using docker-in-docker)
3. deploy DU2 factory and deploy a test DU2

smart-contracts-init has been removed from streamr-docker-dev. It it now used to make preloaded images with `preload_parity_images.sh`

### CODE REVIEW

File: preload_parity_images.sh [hunk 1]
```diff
@@ -0,0 +1,15 @@
+#!/bin/bash -eux
```
Comment (jtakalai) [inline preload_parity_images.sh:1]: Is this called automatically anywhere?

File: README.md [hunk 2]
```diff
@@ -1,5 +1,11 @@
 # Smart Contract init
-Service Will initialize Streamrs Smart Contracts
+Service Will initialize Streamr's Smart Contracts
+
+smart-contracts-init container is no longer part of streamr-docker-dev. Instead we use the docker-compose.yml file here to build the preloaded parity images streamr/open-ethereum-poa-mainchain-preload1 and streamr/open-ethereum-poa-sidechain-preload1.
+
+To build images:
```
Comment (jtakalai) [inline README.md:6]: In what situation would you do this? When building the images for Docker Hub? That process should be automated in a script IMO, we can't rely on hand-built images if there's a need to change/fix something later.

File: README.md [hunk 3]
```diff
@@ -1,5 +1,11 @@
 # Smart Contract init
-Service Will initialize Streamrs Smart Contracts
+Service Will initialize Streamr's Smart Contracts
```
Comment (jtakalai) [inline README.md:2]: What does this mean? Is it a title or a documentation TODO item?

File: README.md [hunk 4]
```diff
@@ -1,5 +1,11 @@
...
+
+smart-contracts-init container is no longer part of streamr-docker-dev. Instead we use the docker-compose.yml file here to build the preloaded parity images streamr/open-ethereum-poa-mainchain-preload1 and streamr/open-ethereum-poa-sidechain-preload1.
+
+To build images:
+./preload_parity_images.sh
+
 
 Parity Default Private Key
```
Comment (jtakalai) [inline README.md:10]: Maybe a little bit of prose here about what all these things are (addresses generated by the preload scripts, common to all streamr-docker-dev setups that use dockers generated from this repository?)

File: README.md (hunk 2 above)
Comment (isentropy) [reply README.md:6]: It is automated by preload_parity_images.sh. I'm not sure how the nightly builds are invoked. @mikhaelsantos ?

File: README.md (hunk 4 above)
Comment (isentropy) [reply README.md:10]: ok

File: preload_parity_images.sh (hunk 1 above)
Comment (isentropy) [reply preload_parity_images.sh:1]: not at the moment. @mikhaelsantos is in charge of the automatic image pushes to dockerhub.

File: README.md (hunk 3 above)
Comment (isentropy) [reply README.md:2]: I'll add more description. This was Mikhael's text

**Comment by isentropy** [comment]
I added some doc to README. Can I merge @jtakalai ?

**Pull request merged**